src/briefcase/__main__.py

```diff
@@ -16,11 +19,17 @@ def main():
         log.error()
         log.error(str(e))
         result = e.error_code
+        trace = extract_stacktrace()
```

Comment:
The exception details are only maintained within the `except` block.....so they have to be gathered here.